stories/topics/proc-aws-deleting-backups-troubleshooting-playbook.adoc

```diff
@@ -1,19 +1,19 @@
 [id="proc-aws-deleting-backups-troubleshooting-playbook"]
 
-= Managing backups playbook
-.Procedure
 
-== Failed to delete a backup
+== Failing to delete a backup
 
 If the deletion of backup fails, take the following steps.
 
-Locate the bucket, or Navigate to the bucket containing the backup.
-Locate the directory that has the name of the backup.
-Open the backup directory
-Open the file names "restore-vars.json" and note last part of the "Recovery_Point_Arn" and the "Backup_Vault_Name"
-navigate to * AWS backup *
-Open Backup Vault and click on the vault with the name you noted from "Backup_Vault_Name"
-Search for the "Recovery_Point_Arn".
-Delete it if not already deleted.
-Go back to the bucket
-Delete the directory having the name of the backup.
\ No newline at end of file
+.Procedure
+. Navigate to the bucket containing the backup.
+. Locate the directory that has the name of the backup.
+. Open the backup directory
+. Open the file names `restore-vars.json` and note last part of the `Recovery_Point_ARN` and the `Backup_Vault_Name`.
+. Navigate to *AWS backup*.
+. Select *Backup Vault*.
```

Comment:
Where. In what field? This is a very vague procedure. It gives very little guidance.

Reply:
We can handle this change external to this PR.
We have a doc validation effort coming up and we will review all of our procedures then to ensure they are working with our release candidate.